lib/fastlane/plugin/wpmreleasetoolkit/helper/user_agent.rb

```diff
@@ -0,0 +1,5 @@
+module Fastlane
+  module Wpmreleasetoolkit
+    USER_AGENT = 'Automattic App Release Automator; https://github.com/wordpress-mobile/release-toolkit/'.freeze
+  end
+end
```

Comment:
I'm not sure whether this is the best way to define a constant scoped to the whole toolkit codebase. I copied this approach from `version.rb`.